statistics.py

```diff
@@ -1,3 +1,55 @@
+import math
 
 def calculateStats(numbers):
-  return None
+
+  keys = ["avg", "max", "min"]
+  vals = []
+
+  if type(numbers) is list:
+
+    if len(numbers) == 0:
+
+      vals = [math.nan for i in keys] # is_nan test
+      # print(vals)
+
+    else:
+
+      vals = [ round((sum(numbers) / len(numbers)), 3), max(numbers), min(numbers) ] # min_max_report test 
+  
+  else:
+
+    return # invalid_arg test
```

Comment:
Took me a while to trace back the `else` here. This function will tend to grow more complex as more conditions are imposed on the inputs and outputs. You can watch out for this by limiting the [cyclomatic complexity](https://www.geeksforgeeks.org/cyclomatic-complexity/). Take a look at that link and we'll talk about tooling related to that quantification later.